code/modules/karma/karma.dm

```diff
@@ -195,6 +182,23 @@ GLOBAL_LIST_EMPTY(karma_spenders)
 	karmashopmenu()
 
 /client/proc/karmashopmenu()
+	var/DBQuery/query = GLOB.dbcon.NewQuery("SELECT * FROM [format_table_name("whitelist")] WHERE ckey='[usr.ckey]'")
```

Comment:
Sanitize the ckey. Yes I know its likely someone will name their account `Robert'); DROP TABLE Students;--` but you can never be so sure

Reply:
Ah yes, little bobby tables, we call him.

Reply:
xkcd never gets old.

Reply:
Sanitation added all round